### PR TITLE
Addresses #2981 NTR 'bronchopulmonary segment'

### LINKED ISSUE #2981: NTR bronchopulmonary segment
**Preferred term label:**
bronchopulmonary segment

**Synonyms**
lung segment PMID:33620799

**Definition (free text, please give PubMed ID)**
A discrete anatomical unit of the lung that is composed of a segmental bronchus and the lung tissue supplied by that bronchus. A bronchopulmonary segment exhibits a pyramidal shape, with its apex pointing towards the center of the lung and its base toward the pleural surface.
[doi:10.1016/j.mpaic.2008.09.022](https://doi.org/10.1016/j.mpaic.2008.09.022)
[PMID:18271171](https://doi.org/10.1016/j.thorsurg.2006.12.008)
[PMID:33620799](https://www.ncbi.nlm.nih.gov/books/NBK567722/#article-24523.s2)
FMA:7312

**Parent term (use https://www.ebi.ac.uk/ols/ontologies/uberon)**
'part of' some 'lobe of lung'

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0001-6677-8489
https://orcid.org/0000-0002-9185-3994

**Comment**
A bronchopulmonary segment is surrounded by connective tissue septas, which are continuous with the pleural surface and prevent air collateralization between segments somewhat. In humans, the right lung has ten bronchopulmonary segments, with eight or nine bronchopulmonary segments in the left lung.
[PMID:18271171](https://doi.org/10.1016/j.thorsurg.2006.12.008)
[doi:10.1016/j.mpaic.2008.09.022](https://doi.org/10.1016/j.mpaic.2008.09.022)

**Child terms**
`'segmental bronchus' SubclassOf overlaps some 'bronchopulmonary segment'`
`'subsegmental bronchus' SubclassOf 'part of' some 'bronchopulmonary segment'`

### PR DESCRIPTION
**Pull request by aleixpuigb**
Fixes #2981 NTR 'bronchopulmonary segment'

### CODE REVIEW

**Comment by aleixpuigb** [comment]
#gogoeditdiff

**Review by bvarner-ebi — CHANGES_REQUESTED** [review]
Please see inline comments.

File: src/ontology/uberon-edit.obo
Comment (aleixpuigb) [reply src/ontology/uberon-edit.obo:224380]: Thank you for your comment. The reason I have added that sentence in the comment is because it does not define the bronchopulmonary segment, but describes structures that surround it.
I will make the edits you propose, thanks!

File: src/ontology/uberon-edit.obo
Comment (aleixpuigb) [reply src/ontology/uberon-edit.obo:224386]: I am still doing some research about what is a pulmonary lobule, as I have seen it defined in different ways. However, according to joshua's comment, the axiom you propose is true.

File: src/ontology/uberon-edit.obo
Comment (bvarner-ebi) [reply src/ontology/uberon-edit.obo:224380]: > is because it does not define the bronchopulmonary segment, but describes structures that surround it.

My understanding is that it defines the boundary, but will defer to requester/SME.

File: src/ontology/uberon-edit.obo
Comment (bvarner-ebi) [reply src/ontology/uberon-edit.obo:224386]: "... the right lung has ten bronchopulmonary segments, and eight or nine bronchopulmonary segments in the left lung."

Recommend removing comma:

"... the right lung has ten bronchopulmonary segments and eight or nine bronchopulmonary segments in the left lung."

**Review by bvarner-ebi — APPROVED** [review]
No breaking changes.
One suggestion to edit before merging [here](https://github.com/obophenotype/uberon/pull/2982/files#r1270397575).

**Pull request merged**
